distributed/scheduler.py

```diff
@@ -4015,31 +4164,136 @@ def __init__(
         self.rpc.allow_offload = False
         self.status = Status.undefined
 
+    ####################
+    # state properties #
+    ####################
+
+    @property
+    def aliases(self):
+        return self.state.aliases
+
+    @property
+    def bandwidth(self):
+        return self.state.bandwidth
+
+    @property
+    def clients(self):
+        return self.state.clients
+
+    @property
+    def extensions(self):
+        return self.state.extensions
```

Comment:
Just like for plugins, I'm wondering how the extensions now are supposed to work (are extensions even something we do publicly? this is merely an internal thing to do, isn't it? Users would use plugins I assume)